chat/ConversationSettings.vue

```diff
@@ -151,7 +173,8 @@
 
         joinMessages: this.joinMessages,
         defaultHighlights: this.defaultHighlights,
-        adSettings: this.conversation.settings.adSettings
+        adSettings: this.conversation.settings.adSettings,
+        enableLogging: this.enableLogging
```

Comment:
Maybe it's better for consistency if this has the same name as the character-level setting. This is the same with other settings that can be overridden per conversation like ``joinMessages``, ``highlight``, ``highlightWords``, etc.